### PR TITLE
Experimental/jspm0.17

### PR DESCRIPTION
**Pull request by ffflabs**
Removes lodash dependency, upgrades jspm to version 0.17 beta.

### CODE REVIEW

File: jspm.config.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+SystemJS.config({
```
Comment (amenadiel) [inline jspm.config.js]: Codacy Issue found: ["SystemJS" is not defined.](https://www.codacy.com/app/amenadiel/ig_markerfactory/file/2450723167/issues/source?bid=3207375&fileBranchId=3248248#l1)

File: jspm.browser.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+SystemJS.config({
```
Comment (amenadiel) [inline jspm.browser.js]: Codacy Issue found: ["SystemJS" is not defined.](https://www.codacy.com/app/amenadiel/ig_markerfactory/file/2450723166/issues/source?bid=3207375&fileBranchId=3248248#l1)

Comment (ffflabs) [inline jspm.browser.js:1]: Codacy Issue found: ["SystemJS" is not defined.](https://www.codacy.com/app/amenadiel/ig_markerfactory/file/2450782131/issues/source?bid=3207375&fileBranchId=3248248#l1)

File: jspm.config.js (hunk 1 above)
Comment (ffflabs) [inline jspm.config.js:1]: Codacy Issue found: ["SystemJS" is not defined.](https://www.codacy.com/app/amenadiel/ig_markerfactory/file/2450782132/issues/source?bid=3207375&fileBranchId=3248248#l1)

**Pull request merged**